static/js/components/CourseList.js

```diff
@@ -88,6 +89,12 @@ class CourseList extends React.Component {
         </div>;
       });
 
+      let applyForMSBtnLabel = `Apply for the ${program.title} Master’s Degree`;
+      const btnStyle = {
+        'text-transform': 'none',
+        'padding': '25px 16px'
+      };
```

Comment:
if i not use text-transform the all text will be capitalize.
On May 28, 2016 12:45 AM, "Peter Pinch" <notifications@github.com> wrote:

> In static/js/components/CourseList.js
> <https://github.com/mitodl/micromasters/pull/411#discussion_r64957696>:
>
> > @@ -88,6 +89,12 @@ class CourseList extends React.Component {
> >          </div>;
> >        });
> >
> > +      let applyForMSBtnLabel = `Apply for the ${program.title} Master’s Degree`;
> > +      const btnStyle = {
> > +        'text-transform': 'none',
> > +        'padding': '25px 16px'
> > +      };
>
> Why is this necessary?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mitodl/micromasters/pull/411/files/2731ca459e94212f32683bda1cf2bc868c4bdb24#r64957696>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ8rErxfjd5E71_ucDW_8l7eE8xGiqb8ks5qF0m_gaJpZM4Imj_g>
> .
>